config/manager/sonataflow_builder_dockerfile.yaml

```diff
@@ -1,5 +1,8 @@
 FROM quay.io/kiegroup/kogito-swf-builder-nightly:latest AS builder
-  
+
+ # variables that can be overridden by the builder
+ARG QUARKUS_EXTENSIONS
```

Comment:
`QUARKUS_EXTENSIONS` is supposed to be an env of `swf-builder` image. Why defining it as a build arg ?
https://github.com/kiegroup/kogito-images/blob/3f454b7b5f040b3e14aa4b65c8af06e3ee81cb27/modules/kogito-swf/common/scripts/module.yaml#L7

Reply:
Cause we can't set an env var in the build process without passing it as an `ARG` to the `Dockerfile`.

Reply:
yeah. seems that build arg definition are not possible in Cekit ...

Reply:
@spolti do u know why not? ^

Reply:
That also means we need to review all var used in those scripts: https://github.com/kiegroup/kogito-images/tree/main/modules/kogito-swf/common/scripts/added
and put them as ARG ...
Not the best

Reply:
No, we might ask in the CeKit channel and propose a new feature.

Reply:
my only point of observation of doing this is, at least using buildah/podman, the build will fail if the ARG in question is not set.

Reply:
This is meant for Kaniko/BuildConfig, and it's working ok. Can u propose on cekit to add `ARG` in the `image.yaml` structure? Maybe with a default value to avoid errors in runtime? 🤔